lib/.eslintrc.json

```diff
@@ -1,68 +1,120 @@
 {
+  "parser": "@typescript-eslint/parser",
   "env": {
     "jest/globals": true,
-    "browser": true,
-    "window": true
+    "browser": true
   },
   "extends": [
     "airbnb",
     "prettier",
     "plugin:jest/recommended",
     "plugin:storybook/recommended",
-    "plugin:storybook/recommended"
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended",
+    // "plugin:security/recommended-legacy",
+    "plugin:react/recommended"
   ],
-  "plugins": [
-    "jest",
-    "react-hooks"
+  "plugins": ["jest", "react-hooks", "@typescript-eslint"],
+  "overrides": [
+    // Test files
+    {
+      "files": ["**/?(*.)+(spec|test).[jt]s?(x)"],
+      "rules": {
+        "jsx-a11y/tabindex-no-positive": "off",
+        "jsx-a11y/no-noninteractive-tabindex": "off",
+        "@typescript-eslint/no-explicit-any": "off"
+      }
+    },
+    // Storybook files
+    {
+      "files": ["**/?(*.)+(stories).[jt]s?(x)"],
+      "rules": {
+        "jsx-a11y/no-noninteractive-tabindex": "off"
+      }
+    },
+    // FocusLock file
+    {
```

Comment:
Maybe we should take a look at FocusLock implementation. Div (Containers in general) should never be focusable.